src/main/resources/db/migration/postgres/V1_320_1__copyCaseNumbersToMediaLinkedCase.sql

```diff
@@ -0,0 +1,20 @@
+DO
```

Comment:
As previously mentioned, would be nice to be made a java flyway class, both for readability and testability but I understand why you havent performed this, so please ignore this comment.